backend/mirisira_api/urls.py

```diff
@@ -1,7 +1,11 @@
 # -*- coding: utf-8 -*-
 from django.conf.urls import url
+
 from mirisira_api.views import *
 
 urlpatterns = [
     url(r'^test$', APITestView.as_view()),
```

Comment:
api-document.yamlに合わせて，testの末尾にも/を入れる必要があるかもしれません（test apiが404で通りませんでした）
```suggestion
    url('test/', APITestView.as_view()),
```